pkg/component/extensions/operatingsystemconfig/original/components/nodeagent/rbac.go

```diff
@@ -104,8 +105,50 @@ func RBACResourcesData(secretNames []string) (map[string][]byte, error) {
 				},
 			},
 		}
+	)
 
-		clusterRoleBindingNodeBootstrapper = &rbacv1.ClusterRoleBinding{
+	if features.DefaultFeatureGate.Enabled(features.NodeAgentAuthorizer) {
+		// When the feature gate is enabled, gardener-node-agent service account needs permissions to create an CSR on
+		// existing nodes for migration.
+		clusterRole.Rules = append(clusterRole.Rules, rbacv1.PolicyRule{
+			APIGroups: []string{"certificates.k8s.io"},
+			Resources: []string{"certificatesigningrequests"},
+			Verbs:     []string{"create", "get"},
+		})
+	} else {
+		// For the case that NodeAgentAuthorizer feature gate is disabled again node-agents group have to be added to
+		// (cluster) role binding that node-agents which are already migrated do not lose access.
```

Comment:
```suggestion
		// For the case that NodeAgentAuthorizer feature gate is disabled again node-agents group has to be added to
		// (cluster) role binding so that node-agents which are already migrated do not lose access.
```